### PR TITLE
Feature/logging/initial commit

### PR DESCRIPTION
**Pull request by andreicoval**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+Uploading
+
+Deploying with Maven
+
+In Maven’s setting.xml file, add the following section to declare your Bintray credentials. Use your API key as your password (not your login password):
+    
+    <servers>
+            <server>
+                <id>bintray-smart-home-oss-m2</id>
+                <username>"username"</username>
```
Comment (rodislav) [inline README.md:72]: remove the quotes pls, it can be missleading

File: README.md [hunk 2]
```diff
@@ -1 +1,85 @@
...
+                <password>"password"</password>
```
Comment (rodislav) [inline README.md:73]: here as well

File: README.md [hunk 3]
```diff
@@ -1 +1,85 @@
-# logging
\ No newline at end of file
+# logging
+
+Downloading
+
+Resolving artifacts using Maven
+
+To download "logging" artifact, add below profiles to settings.xml file, located in ~/.m2 folder:
+<profiles>
```
Comment (rodislav) [inline README.md:8]: please enclose the code examples in special backtick tags
https://guides.github.com/features/mastering-markdown/#examples  -  Code

**Comment by andreicoval** [comment]
Done

**Pull request merged**